### PR TITLE
added custom frame filter

### LINKED ISSUE #610: How to create a tile image from a video?
I'm using ffmpeg with the following command for a spritemap creation:

`ffmpeg -i input.mp4 -vf 'scale=142:80,fps=1,tile=10x10:margin=2:padding=2' output.jpg`

I've already tried to use the CustomFilter to create it within PHP-FFmpeg, but it lacks of saving image formats. What could be done to save the filtered tile image?

### PR DESCRIPTION
**Pull request by iljalukin**
| Q                  | A
| ------------------ | ---
| Bug fix?           | no
| New feature?       | yes
| BC breaks?         | no
| Deprecations?      | no
| Fixed tickets      | fixes #610 
| Related issues/PRs | #610 
| License            | MIT

#### What's in this PR?

This adds a custom filter to frame

#### Why?

E.g. it's possible to create image tiles from a video, see example code.

#### Example Usage

```php
$ffmpeg = FFMpeg\FFMpeg::create();
$video = $ffmpeg->open('/some/video.mp4');
$video->frame(FFMpeg\Coordinate\TimeCode::fromSeconds(0))
      ->addFilter(new CustomFrameFilter('scale=142:80,fps=1,tile=10x10:margin=2:padding=2'))
      ->save('/some/picture.jpg');

### CODE REVIEW

**Comment by jens1o** [comment]
Nice contribution. Thanks!

One more thing: Could you please add a test? Just like https://github.com/PHP-FFMpeg/PHP-FFMpeg/blob/6dacd82c9b1c86e67563340d509b500815144178/tests/Unit/Filters/Video/SynchronizeFilterTest.php

File: tests/Unit/Filters/Frame/CustomFrameFilterTest.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+use FFMpeg\Filters\Frame\CustomFrameFilter;
+use Tests\FFMpeg\Unit\TestCase;
+
+class CustomFrameFilterTest extends TestCase
+{
+    public function testApplyCustomFrameFilter()
+    {
+	$frame = $this->getFrameMock();
+
+	$filter = new CustomFrameFilter('whatever i put would end up as a filter');
```
Comment (jens1o) [inline tests/Unit/Filters/Frame/CustomFrameFilterTest.php]: Please fix the formatting(fix intending).

Comment (iljalukin) [reply tests/Unit/Filters/Frame/CustomFrameFilterTest.php]: Done. Sorry for inconvenience.

**Comment by jens1o** [comment]
I went ahead and changed the tabs to spaces, to be compliant with the rest of the codebase. As soon as travis passed, I'm happy to merge. Thanks again :)

**Pull request merged**